### ISSUE TITLE
Should this work with a react app?

### ISSUE THREAD

**Issue by bilthon** [kurdin/console-remote#8]
I have this `create-react-app` and was wondering if this solution would work on it.

**Comment by kurdin** [comment]
Yeah, I made some changes, now you can do this in your react or any app with webpack or other bundler:

do
`npm i console-remote-client`

change your index.js
```js
import React from 'react';
import { render } from 'react-dom';
import MyAppTodo from './MyAppTodo';
import registerServiceWorker from './registerServiceWorker';

import 'console-remote-client';

console.re.settings({
  channel: 'my-todo-logs'
});
// view remote logs at https://console.re/my-todo-logs

render(<AppTodo />, document.getElementById('root'));
registerServiceWorker();
```

visit `https://console.re/my-todo-logs` 
now you can load your app in browser and do: `console.re.log('testing')`

**Comment by devdaddy** [comment]
Anyway you have an example for loading into angular project?  I already have it installed, I am having trouble getting it configured for my project.

**Comment by mhs1** [comment]
Very helpfull. Please add info to "How to connect" on website! 

![image](https://user-images.githubusercontent.com/46316862/73166647-84101380-40f6-11ea-8da9-ab46e5d21804.png)

**Comment by kurdin** [comment]
@mhs1 Yes, it is in there already, just called `How to Install`
<img width="756" alt="Screen Shot 2020-01-27 at 8 20 11 AM" src="https://user-images.githubusercontent.com/6027060/73178014-65058780-40de-11ea-860a-176c7b92e67b.png">
